requirements.txt

```diff
@@ -1,2 +1,3 @@
 Jinja2==2.8
+Django==1.11.4
```

Comment:
Yeah I have removed the requirements. 
Perhaps there should also be a documentation that states this